discord/ui/select.py

```diff
@@ -97,6 +111,82 @@
 selected_values: ContextVar[Dict[str, List[PossibleValue]]] = ContextVar('selected_values')
 
 
+def _handle_select_defaults(
+    defaults: Sequence[ValidDefaultValues],
+    component_type: Literal[
+        ComponentType.user_select,
+        ComponentType.role_select,
+        ComponentType.channel_select,
+        ComponentType.mentionable_select,
+    ],
+) -> List[SelectDefaultValue]:
+    if not defaults:
+        return []
+
+    if component_type not in (
+        ComponentType.user_select,
+        ComponentType.role_select,
+        ComponentType.channel_select,
+        ComponentType.mentionable_select,
+    ):
+        raise TypeError('component_type must be one of user_select, role_select, channel_select, or mentionable_select')
```

Comment:
I don't quite get the point of this check in an internal function?

Reply:
Thought it could prevent some sort of KeyError when being called by... a user but yeah it shouldn't be called at all outside the lib. Removing.